FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -20,5 +20,14 @@ public final class RemoteFeedLoader: FeedLoader {
 	
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
 		
+		client.get(from: url) { [weak self] (result) in
+			guard self != nil else { return }
+			switch result {
+			case .success((let data, let response)):
+				completion(Root.feedImagesFromData(from: data, stausCode: response.statusCode))
```

Comment:
There is a typo on the argument label `stausCode`, there's a missing "t". Please rename the function and follow the build failures to update any other places using the parameter.

```suggestion
				completion(Root.feedImagesFromData(from: data, statusCode: response.statusCode))
```